src/Goerli-DssSpell.sol

```diff
@@ -21,12 +21,16 @@ pragma solidity 0.6.12;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-contract DssSpellAction is DssAction, DssSpellCollateralAction {
+import { DssSpellCollateralAction } from "./Goerli-DssSpellCollateral.sol";
 
+contract DssSpellAction is DssAction, DssSpellCollateralAction {
     // Provides a descriptive tag for bot consumption
     string public constant override description = "Goerli Spell";
 
-    uint256 internal constant MILLION = 10 ** 6;
+    // Turn office hours off
+    function officeHours() public override returns (bool) {
+        return false;
+    }
```

Comment:
I kind of think this should be below the internal constants so all the constants are together and the functions are together

Reply:
I copied [this](https://github.com/makerdao/spells-goerli/blob/PE-1098-2022w41/template/Goerli-DssSpell.sol) per the checklist for this :)